x/staking/README.md

```diff
@@ -114,6 +114,10 @@ Validators can have one of three statuses
   tombstoning, an unbonding of all their delegations begins. All delegations must then wait the UnbondingTime
   before their tokens are moved to their accounts from the `BondedPool`.
 
+:::Note
```

Comment:
This won't render nice. It had to be `::: note`.
![image](https://user-images.githubusercontent.com/29894366/227582641-c8926671-a60b-4c41-a2ab-5a4956546837.png)